library/core/functions.general.php

```diff
@@ -3110,6 +3110,32 @@ function safeImage($ImageUrl, $MinHeight = 0, $MinWidth = 0) {
     }
 }
 
```

Comment:
That is really confusing.
The doc block says URL.

And that function seems dangerous:
```php
print_r([
    matchesTrustedDomain('http://vanilla.*'), // false,
    matchesTrustedDomain('vanilla.*'), // true
    matchesTrustedDomain('www.somethingelse.*'), // true
    matchesTrustedDomain('literallyanything.com/somethingwithoutavalidscheme'), // true
    matchesTrustedDomain('*'), // true
])
```